Observing-Interface-Setup/nublado-notebook-environment-update.rst

```diff
@@ -72,7 +74,8 @@ Post-Condition
 .. It is preferred to include them as a bulleted or enumerated list.
 .. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
 
-- The requested software version(s) is deployed for use in the jupyter-lab notebook. This will over-ride the versions available on the server.
+- The requested software version(s) is deployed for use in the jupyter-lab notebook.
+  This will over-ride the versions available on the server.
```

Comment:
maybe mention it doesn't over-write them? Meaning they can just remove the setup lines etc and revert back to the base container packages?